tests/test_api.py

```diff
@@ -0,0 +1,140 @@
+import pytest
+from fastapi.encoders import jsonable_encoder
+from fastapi.testclient import TestClient
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.pool import StaticPool
+
+from app import crud
+from app.api import app, get_db
+from app.db import Base
+from app.schemas import PriceCreate, UserBase
+
+# database setup
+# ------------------------------------------------------------------------------
+SQLALCHEMY_DATABASE_URL = "sqlite://"
+
+engine = create_engine(
+    SQLALCHEMY_DATABASE_URL,
+    connect_args={"check_same_thread": False},
+    poolclass=StaticPool,
+)
+TestingSessionLocal = sessionmaker(autocommit=False, autoflush=False, bind=engine)
+
+Base.metadata.create_all(bind=engine)
```

Comment:
Isn't it better to do DB initialization in a global (autouse=True, scope=session) fixture? So that we can take care of removing it after

Reply:
I honestly don't know, this is more or less a copy-paste of the fastapi tutorials
https://fastapi.tiangolo.com/advanced/testing-database/

Reply:
I'll merge, but these tests will need improvement for sure !